src/AddListItem.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Button from './Button.js';
+
+export default function AddListItem({ onAddItem }) {
+  return (
+    <Form
+      aria-label="Add a dive destination to your wishlist"
+      onSubmit={handleAddItem}
+    >
+      <Container>
+        <Label htmlFor="destination">DESTINATION </Label>
+        <Input id="destination" name="destination" required />
+      </Container>
+      <Container>
+        <Label htmlFor="notes">NOTES</Label>
+        <Textarea id="notes" name="notes" required />
+      </Container>
+      <Button type="sumit">Add to list</Button>
```

Comment:
little correction -> type="submit"
still works because its a type - submit by default

Reply:
Hups, thanks a lot :) Didn't see that!